evansmusomi/react-native-102
# Updates redux for tabbed navigation

*What this does*
- Connects redux to tabbed navigation screens
- Updates redux to work with find and share places functionality

*How to verify*
- Press the Share Place tab
- Enter a place name in the input field and press the Add Place button
- Navigate to the Find Place tab
- The places you shared should appear as list items on this screen
